apps/dashboard/src/components/InputOrganSpan.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <div>
+    <span :class="['flex flex-wrap justify-content-between',
+     column ? 'flex-column gap-1' : 'flex-row align-items-center gap-3']">
+      <span class="my-0">{{ label }}</span>
+      <Dropdown
+          v-model="organ"
+          :options="organs"
+          optionLabel="firstName"
+          :placeholder="placeholder"
+          class="w-full md:w-15rem"
+      >
+    <template #option="slotProps">
+      {{ slotProps.option.firstName + " " + slotProps.option.lastName }}
```

Comment:
NIT: extract to a function in the script body that also checks if there is a lastName?

```
function fullName(user: BaseUserResponse) {
   return user.lastName ? : user.FirstName + ' ' + user.Lastname : user.Firstname
}
```

Reply:
Done!